public/scss/_variables.scss

```diff
@@ -13,6 +13,7 @@ $color-red-light: #f3e9e8;
 $color-purple: #7834bc;
 $color-grey-light: #cbcbcb;
 $color-grey-dark: #666666;
+$stepper-grey-light: #6f777b;
```

Comment:
Visually, this is _very_ close to `666666`. Can we just use that one instead? (having lots of similar colours is the thing that variables are supposed to address).

Reply:
Or we could change `$color-grey-dark` to `6f777b ` if that's a better solution.